report-std-change.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
```

Comment:
Long bash scripts scare me but as long as it stays around this size, I’m fine with it :slightly_smiling_face: